src/ripple/app/main/LoadManager.cpp

```diff
@@ -32,40 +33,56 @@
 
 namespace ripple {
 
-class LoadManagerImp
-    : public LoadManager
-    , public beast::Thread
+class LoadManagerImp : public LoadManager
 {
 public:
-    using LockType = std::mutex;
-    using ScopedLockType = std::lock_guard <LockType>;
+    //--------------------------------------------------------------------------
 
-    Application& app_;
-    beast::Journal m_journal;
-    LockType mLock;
+    Application& mApp;
+    beast::Journal mJournal;
```

Comment:
Consistency is not the most important goal. I would have simply adopted the trailing underscore only for newly added data members as is our convention (allowing readers to visually distinguish old code from new code). I could also live with updating the pre-existing data members to our convention of a trailing underscore, using the argument that there were logic changes. But I don't see utility in updating the entire file to use an old convention, for the "consistency" canard.